Modules/Process/Get-PrefetchFiles.ps1

```diff
@@ -1,13 +1,38 @@
 ﻿<#
 .SYNOPSIS
 Acquires copies of prefetch files, if enabled, copying them
-to a zip file. This is not the fastest, but in my testing, it
-works.
+to a zip file. The zip file is then encoded and returned as a powershell object. 
 .NOTES
+The output of this script needs to be processed by the Deserialize-KansaField.ps1 script.
+Example:
+Deserialize-KansaField.ps1 -InputFile .\COMPUTERNAME-PrefetchFiles.csv -Field Content -OutputFile PrefetchFiles.zip
 Next line is Kansa.ps1's output directive
-OUTPUT zip
+OUTPUT csv
```

Comment:
Output directives are no longer a thing. Everything is output in the format specified via the -OutputFormat parameter. See https://trustedsignal.blogspot.com/2015/08/kansas-stafford-release-more-capable.html for details. Those lines can be removed entirely.